src/data/qpf/univariate/basic.lean

```diff
@@ -0,0 +1,186 @@
+/-
+Copyright (c) 2018 Jeremy Avigad. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Jeremy Avigad
+
+Polynomial functors. Also expresses the W-type construction as a polynomial functor.
+(For the M-type construction, see Mtype.lean.)
+-/
+import tactic.interactive data.multiset
+universe u
+
+/- TODO (Jeremy): move this. -/
```

Comment:
I think this is your todo now, Simon. :smile:

Reply:
Good catch :)